### PR TITLE
Feature/manual model creation plugin

### LINKED ISSUE #7: Add Implementation for ManualModelCreationPlugin
This plugin should support use-cases in which the system admin enters information about the instance model manually via Winery because no IaC technology for deployment management is being used.

### PR DESCRIPTION
**Pull request by ghareeb-falazi**
The plugin allows configuring a `modelPath` from which an EDMM-based instance model will be retrieved (using HTTP GET)

closes #7 
builds upon #6

### CODE REVIEW

File: src/main/java/org/iac2/service/checking/plugin/implementation/subgraphmatching/RuleChecker.java [hunk 1]
```diff
@@ -1,17 +1,18 @@
 package org.iac2.service.checking.plugin.implementation.subgraphmatching;
 
+import java.util.Iterator;
+
 import io.github.edmm.model.component.RootComponent;
 import io.github.edmm.model.relation.RootRelation;
 import org.iac2.common.model.InstanceModel;
 import org.iac2.common.model.compliancerule.ComplianceRule;
+import org.iac2.common.utility.EdmmGraphCreator;
```
Comment (nyuuyn) [inline src/main/java/org/iac2/service/checking/plugin/implementation/subgraphmatching/RuleChecker.java:9]: This import is new but doesn't seem to be used in the class ? Atleast from the github perspective

File: src/test/java/org/iac2/service/architecturereconstruction/plugin/implementation/manual/ManualModelCreationPluginTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        plugin.setConfigurationEntry(ManualModelCreationPlugin.CONFIG_ENTRY_MODEL_PATH, path);
+        InstanceModel result = plugin.reconstructInstanceModel(
+                new ProductionSystem("none of your business", "saaa", new HashMap<>()));
+        Assertions.assertNotNull(result);
+        Assertions.assertNotNull(result.getDeploymentModel());
+        Assertions.assertEquals(3, result.getDeploymentModel().getComponents().size());
+    }
+
+    @Test
+    void testTheObvious() {
```
Comment (nyuuyn) [inline src/test/java/org/iac2/service/architecturereconstruction/plugin/implementation/manual/ManualModelCreationPluginTest.java:27]: 🙈

**Review by nyuuyn — APPROVED** [review]
LGTM

File: src/main/java/org/iac2/service/checking/plugin/implementation/subgraphmatching/RuleChecker.java (hunk 1 above)
Comment (ghareeb-falazi) [reply src/main/java/org/iac2/service/checking/plugin/implementation/subgraphmatching/RuleChecker.java:9]: I have no idea how this happened. It is used in the constructor. So not sure why that usage is not new, or how this import is new :\

**Pull request merged**